### DISCUSSION TITLE
更新修正时区 8点后又会多一天

### DISCUSSION THREAD

**Discussion by AprDeci** [wangxinleo/wechat-public-account-push#72] [General]
rt

**Comment by wangxinleo** [comment]
![mmexport1661994696492.png](https://user-images.githubusercontent.com/28293988/187811448-09f386f8-ddfe-4181-83a7-99a8167374e3.png)

**Comment by wangxinleo** [comment]
不会吧？是我发现以前的组件对于天数会进行向下取整，导致了还剩0.x天的时候会变成0天，究其原因还是因为utc对于prc时区的0时为16时。

即今天9月1日，若某节日为9月2日时，到下午16点后会判定为已到达9月2日，造成提醒错误。

由此特意更正了。

可以再仔细测测看。

**Comment by wangxinleo** [comment]
> 时间会比之前多一天。（如题）

不好意思，太急了，一不小心把你的帖子编辑了。我错啦💔💔💔💔
